lib/util/ignore.ts

```diff
@@ -3,7 +3,7 @@ import { regEx } from './regex';
 
 export function isSkipComment(comment?: string): boolean {
   if (regEx(/^(renovate|pyup):/).test(comment)) {
-    // TODO #12070 #12071 needs to be checked manually
+    // TODO #12071
```

Comment:
```suggestion
```

Reply:
If we export a function and if the function is inside a for loop in the file it is being imported to, will the regex be created for each loop or it won't be?

Reply:
nope, the `regEx` function will always return the same instance for same input. Thats how our simple cache works.